arguebuf/load/_load_aif.py

```diff
@@ -16,17 +17,230 @@
 __all__ = ("load_aif",)
 
 
+class NewNode:
+    def __init__(self, node_id, text, node_type, timestamp):
+        self.node_id = node_id
+        self.text = text
+        self.node_type = node_type
+        self.timestamp = timestamp
+        self.incoming_edges = []
+        self.outgoing_edges = []
+
+    def add_incoming_edge(self, edge):
+        self.incoming_edges.append(edge)
+
+    def add_outgoing_edge(self, edge):
+        self.outgoing_edges.append(edge)
+
+    def __repr__(self):
+        return f"Node(node_id={self.node_id}, text={self.text}, type={self.node_type}, timestamp={self.timestamp}, incoming_edges={self.incoming_edges}, outgoing_edges={self.outgoing_edges})"
+
+
+def get_connected_nodes_of_type(node, node_type, nodes):
```

Comment:
Please add the correct types here (and also for all following function definitions).